tests/fixtures/repo_with_token_url.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
:thinking: For some reason, this test failed locally with `/bin/sh` for me (yet other tests passed :shrug:). Some quick googling said that `set -eu -o pipefail` is a bashism.